assets/sass/components/_prisoner-profile.scss

```diff
@@ -30,6 +30,7 @@
 
   color: white;
   margin-right: govuk-spacing(2);
+  max-width: unset;
```

Comment:
GOV.UK Tag now has a default `max-width`; removing as it doesn't work well with our use of it.